ARROW-17322: [Docs] Documenting issue lifecycle for bugs and feature requests (#13781)
apache/arrow: docs/source/developers/guide/step_by_step/finding_issues.rst

the top to do the same.

You are ready to create the issue! Add a title and a description following
the guidance in :ref:`creating issues <creating-issues>` and you are ready to go!

.. seealso::
	:ref:`Creating issues <creating-issues>`

> You don’t need any special permissions on JIRA to be able to create issues.
> Once you are more involved in the project and want to do more on JIRA, for
> example assigning yourself an issue, you will need **“Contributor” permissions**.

Reviewer: ```suggestion
You don’t need any special permissions on JIRA to be able to create
or self-assign issues.
```
